guide/src/tutorials/osc/osc-sender.md

```diff
@@ -0,0 +1,155 @@
+
+```rust
+let args = vec![osc::Type::Float(x), osc::Type::Float(y)];
+```
+
+Now, bringing these two things together we get an OSC packet. The sender expect these to be delivered in a tuple.
+
+```rust
+let packet = (osc_addr, args);
+```
+
+[Reading the documentation](https://docs.rs/nannou_osc/latest/nannou_osc/send/struct.Sender.html#method.send-1) for the `send`-method, we can see that it returns a Result type which will either contain the number of bytes written (if it was successful) and, more importantly, some useful errors of type CommunicationError if it was not succesful. To discard the error part of this, we use the `ok()` method at the end. 
+
+```rust
+model.sender.send(packet).ok()
+```
+
+## The finished app
+
+```rust
+use nannou::prelude::*;
+use nannou_osc as osc;
+
+fn main() {
+    nannou::app(model).update(update).simple_window(view).run();
+}
+
+struct Model {
+    sender: osc::Sender<osc::Connected>,
+}
+
+fn model(_app: &App) -> Model {
+    // The network port that data is being sent to
+    let port = 1234;
+
+    // The osc-sender expects a string in the format "address:port", for example "127.0.0.1:1234"
+    // "127.0.0.1" is equivalent to your computers internal address.
+    let target_addr = format!("{}:{}", "127.0.0.1", port);
+
+    // This is the osc Sender which contains a couple of expectations in case something goes wrong.
+    let sender = osc::sender()
+        .expect("Could not bind to default socket")
+        .connect(target_addr)
+        .expect("Could not connect to socket at address");
+
+    Model { sender }
+}
+
+fn update(_app: &App, _model: &mut Model, _update: Update) {}
+
+fn view(app: &App, model: &Model, frame: Frame) {
+    // Use app time to progress through a sine wave
+    let sine = app.time.sin();
+    let slowersine = (app.time / 2.0).sin();
+
+    // Get boundary of the window (to constrain the movements of our circle)
+    let boundary = app.window_rect();
+
+    // Map the sine wave functions to ranges between the boundaries of the window
+    let x = map_range(sine, -1.0, 1.0, boundary.left(), boundary.right());
+    let y = map_range(slowersine, -1.0, 1.0, boundary.bottom(), boundary.top());
+
+    // Send x-y coordinates as OSC
+    let osc_addr = "/circle/position".to_string();
+    let args = vec![osc::Type::Float(x), osc::Type::Float(y)];
+    let packet = (osc_addr, args);
+
+    model.sender.send(packet).ok();
+
+    // Prepare to draw.
+    let draw = app.draw();
+
+    // Clear the background to purple.
+    draw.background().color(PLUM);
+
+    // Draw a blue ellipse at the x/y coordinates 0.0, 0.0
+    draw.ellipse().color(STEELBLUE).x_y(x, y);
+
+    draw.to_frame(app, &frame).unwrap();
+}
+```
```

Comment:
Maybe this could end with something like:

> In the [next tutorial](<link-to-osc-sender-tute>) we will take a look at how to receive our OSC values and do something interesting with them.